context-free.md

```diff
@@ -2,21 +2,65 @@
 
 ##  Who is this programming language for?
 
+Context Free if for people who want to quickly make random and mathematically
+inspired animations and pictures. (Although it may not be entirely intended to
+make animations, the slow renderer combined with the time commands make it a
+possible use case.)
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to make pattern filled, recursive, and random images due to the
+nature of the language. The loop syntax is especially helpful in making
+interesting patterns.
+
+It is also easy, no matter the end picture, to understand the composobility of
+shapes if the code chunks are factored appropriately to different "shape" names.
+Since we can nest primitive and custom shapes within one another, it makes it
+easier to preciesly refer to a part of the picture should someone need to give
+feedback or design critiques.
 
 ## What is hard to do in this language? Why is it hard?
 
+It is hard to create pictures of non-organic items (or perhaps my artistic
+skills and familiarity with the language need more developing). Since
+ContextFree is code that starts with primitive shapes, one must try really hard
+to mentally divide a non-organic shape into primitive shapes if they want to
+draw it accurately. Making adjustments requires a bit of guess and check then
+a slow re-render which makes it harder.
 
```

Comment:
I think there is definitely a relationship between limited expressiveness and DSL-ness, but I could imagine a point where such limited expressiveness makes the language no longer a language&mdash;let alone a DSL. At this point, it would be more like a configuration file.

tl;dr I think there is a **limited** relationship between DSL-ness and the limit of the language.